### PR TITLE
Update assertions to add more detail

### PR DESCRIPTION
**Pull request by bnbarham**
`testSystemSwiftInterface` is called multiple times in a single test, add file/line to the test output so it's clear which case actually failed.

### CODE REVIEW

**Comment by bnbarham** [comment]
@swift-ci please test

File: Tests/SourceKitLSPTests/SwiftInterfaceTests.swift [hunk 1]
```diff
@@ -114,7 +114,9 @@ final class SwiftInterfaceTests: XCTestCase {
     _ testLoc: TestLocation, 
     ws: SKSwiftPMTestWorkspace, 
     swiftInterfaceFile: String, 
-    linePrefix: String
+    linePrefix: String,
+    file: StaticString = #filePath,
+    line: UInt = #line
```
Comment (benlangmuir) [inline Tests/SourceKitLSPTests/SwiftInterfaceTests.swift:119]: There are more XCT assertions here that should probably take line/file:
* XCTFail in the guard else
* XCTUnwrap called twice below

**Comment by bnbarham** [comment]
@swift-ci please test

**Comment by bnbarham** [comment]
@swift-ci please test Windows platform

**Comment by ahoppen** [comment]
We haven’t seen this failure ever since we stopped using the system-wide module cache.

**Pull request closed without merging**